### PR TITLE
MH-12957 Defaults on tab Source in Add Event wizards are broken

### PR DESCRIPTION
**Pull request by staubesv**

### CODE REVIEW

**Comment by staubesv** [comment]
Hi @slampunk 

As you are participating the Comitter Mentorship program, I've found some work for you:

Our supplier has broken the defaults that are set in the Add Event wizard (sorry for that). Admittedly, the naming of the properties and variables (end time, endslot) somewhat suggested that those might be related to the end time of an recordings (which they are in no way).

Anyway, as that feature originated from you, could you please review this PR?

**Comment by slampunk** [comment]
Hi @staubesv 

I'll be happy to review. I haven't done a review on Github yet for our project... would someone assign me as the reviewer?

P.S variable names were somewhat ambiguous, so rather let me say: sorry for that :)

**Comment by staubesv** [comment]
Hi @slampunk 

For whatever reason, I cannot assign this pull request to you. I've assigned it to @rrolf as he is your official mentor.

Thanks for taking the review!

btw: Note that I haven't adjusted the variable names in the configuration file by intention as this goes to 5.x and I assume that people expect minor updates to be as less problematic as possible.

File: modules/admin-ui/src/main/webapp/scripts/shared/services/wizards/new-event/source.js [hunk 1]
```diff
@@ -489,24 +489,27 @@ angular.module('adminNg.services')
...
+                    // Find the next available timeslot for an event's start time
                     if (timeDiff > 0) {
                         var multiple = Math.ceil( timeDiff/(intervalMins * 60) );
                         chosenSlot.add(multiple * intervalMins, 'minute');
                         if (chosenSlot.unix() >= endSlot.unix()) {
-                            endSlot = moment( chosenSlot ).add(durationMins, 'minutes');
+                          // The slot would start after the defined cutoff time (too late in the day), so we
+                          // use the day's start time on tomorrow
+                          chosenSlot = moment( moment().format('YYYY-MM-DD') + ' ' + startTime ).add(1, 'day');
+
```
Comment (slampunk) [inline modules/admin-ui/src/main/webapp/scripts/shared/services/wizards/new-event/source.js]: Unnecessary white space?

**Comment by slampunk** [comment]
LGTM, can be merged

File: modules/admin-ui/src/main/webapp/scripts/shared/services/wizards/new-event/source.js (hunk 1 above)
Comment (staubesv) [reply modules/admin-ui/src/main/webapp/scripts/shared/services/wizards/new-event/source.js]: Fixed.

**Pull request merged**